src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -211,6 +279,49 @@ export const SettingsPane: FunctionComponent = () => {
             </div>
           </div>
         )}
+        {userContext.apiType === "SQL" && (
+          <div className="settingsSection">
+            <div className="settingsSectionPart">
+              <div>
+                <legend id="queryTimeoutLabel" className="settingsSectionLabel legendLabel">
+                  Query Timeout
+                </legend>
+                <InfoTooltip>
+                  When a query reaches a specified time limit, a popup with an option to cancel the query will show
```

Comment:
Let's update this to add "unless automatic cancellation has been enabled" or something like that.